vizro-core/examples/visual-vocabulary/pages/ranking.py

```diff
@@ -79,5 +80,6 @@
     ],
 )
 
+lollipop = lollipop_factory("ranking")
```

Comment:
As it stands this plot is not ordered so doesn't work well in this section. I think we should handle it the same way we handle ordered bar/column where we have a `sort` for those plots. This would also mean it's not actually a chart factory after all I think. wdyt @huong-li-nguyen?

Reply:
I would probably create a sorted one and use that factory function for now (given that we don't have different svgs for the lollipop at the moment). We can always change that later on. So for now:

- Create a sorted lollipop chart
- Leave the factory function here as is 👍

Reply:
@huong-li-nguyen so that would mean the factory function and example.py needs to be the sorted version, right? Like this I think:

```
lollipop(gapminder.query("year == 2007 and gdpPercap > 36000").sort("gdpPercap"), "country", "gdpPercap")
```